src/kalastatic.js

```diff
@@ -1,9 +1,10 @@
 'use strict'
-var assert = require('assert')
-var path = require('path')
-var kss = require('kss/lib/cli')
-var Metalsmith = require('metalsmith')
-var extend = require('extend')
+
+const assert = require('assert')
+const path = require('path')
+const kss = require('kss/lib/cli')
+const Metalsmith = require('metalsmith')
+const extend = require('extend')
```

Comment:
Looks like `import` will only work in Node.js harmony mode. Let's keep it out for now.